### DISCUSSION TITLE
How to set table of contents collapsed by default

### DISCUSSION THREAD

**Discussion by cgutierr-zgz** [jpanther/congo#346] [Q&A]
Hi,  I'd like to be able, if possible to: set the table of contents of a given post to be collapsed by default.

**Comment by jpanther** [comment]
This isn't something that is directly configurable however you could override the `toc.html` partial to remove the "open" from the first line which will prevent it from being automatically opened.

**Reply by cgutierr-zgz** [reply]
That would delete it from fullscreen too but nvm it looks good anyway being opened by default, thanks!
